migrator/models.py

```diff
@@ -85,3 +85,54 @@ class CdnCertificate(CdnBase):
     # certificate is the actual body of the certificate chain
     certificate = sa.Column(postgresql.BYTEA)
     expires = sa.Column(postgresql.TIMESTAMP)
+
+
+class CustomDomainRoute(CustomDomainBase):
+    __tablename__ = "routes"
+
+    guid = sa.Column(sa.Text, primary_key=True)
+    state = sa.Column(sa.Text)
+    domains = sa.Column(sa.Text)
+    challenge_json = sa.Column(postgresql.BYTEA)
+    user_data_id = sa.Column(sa.Integer)
+    alb_proxy_arn = sa.Column(sa.Text)
+    certificates = orm.relationship("CustomDomainCertificate")
```

Comment:
No need/desire for [`back_populates`](https://docs.sqlalchemy.org/en/14/orm/relationship_api.html#sqlalchemy.orm.relationship.params.back_populates) here?  Totally fine if not, was just curious.

Reply:
I think that's handled on 110